### PR TITLE
Conformance results for v1.20/k0s v0.11.0

### PR DESCRIPTION
**Pull request by trawler**
Signed-off-by: Karen Almog <kalmog@mirantis.com>

### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form)?
* [x] Did you include the product/project logo in SVG, EPS or AI format?  
* [x] Does your logo clearly state the name of the product/project and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] If your product/project is open source, did you include the `repo_url`?
* [x] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

### CODE REVIEW

**Comment by jnummelin** [comment]
@ErinThacker I see you have been merging other PRs lately, could you take a look at getting this merged too?

We're about to submit an update for version infos in next few days so would be nice to get this out of the way first. :)

**Comment by ErinThacker** [comment]
Looking at it now!

File: v1.20/k0s/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Full instructions on how to set up a k0s cluster can be found [here](https://git
 k0s runs either as a single node (server with `--enable-worker`), or as a controller/worker cluster.
 These instructions assume you have one or more linux or arm boxes ready for installation.
 
-Download the k0s v0.10.0 binary from [releases](https://github.com/k0sproject/k0s/releases/v0.10.0) and push it to all the nodes you wish to connect to the cluster.
+Download the k0s v0.11.0 binary from [releases](https://github.com/k0sproject/k0s/releases/v0.11.0) and push it to all the nodes you wish to connect to the cluster.
```
[the remark this answers is not in the document]
Comment (jnummelin) [reply v1.20/k0s/README.md:9]: There is instructions few lines below how to set it up

[the remark this answers is not in the document]
Comment (ErinThacker) [reply v1.20/k0s/README.md:9]: We require more information. Detailed, step-by-step instructions for installation instructions. Referring to your GitHub repo will not suffice. Let us know if you need further guidance.

**Comment by amye** [comment]
Oh, this is fine. 
https://github.com/cncf/k8s-conformance/tree/master/v1.20/k0s and https://github.com/cncf/k8s-conformance/tree/master/v1.19/k0s are similar, all is well.

**Comment by amye** [comment]
You are now Certified Kubernetes

**Pull request merged**